ui/app/peers/create/page.tsx

```diff
@@ -27,13 +39,25 @@ export default function CreatePeer() {
               Data source
             </Label>
           }
-          action={<Select placeholder='Select' id='source' />}
+          action={
+            <SelectSource peerType={peerType} setPeerType={setPeerType} />
+          }
         />
       </Panel>
       <Panel>
         <ButtonGroup>
-          <Button>Cancel</Button>
-          <Button variant='normalSolid'>Continue</Button>
+          <Button as={Link} href='/peers'>
+            Cancel
+          </Button>
+          <Button
```

Comment:
This is confusing we should block at Data source selection only or grey out the unavailable datasource types

![image](https://github.com/PeerDB-io/peerdb/assets/39798643/48ef9c4d-e80a-4038-b428-b10b7dbe0ad4)

Reply:
Yup. Singled out postgres (for now) as the only option in the data source selection step.